test/utils.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+const crypto = require('crypto')
+
+module.exports = {
+  getSubnet: function () {
+    return Math.floor(Math.random() * 255)
+  },
```

Comment:
This seems to duplicate the `getSubnet` inner function defined in `randomIp` below?